docs/content/mobile/ios/local-deploy.md

```diff
@@ -63,5 +63,16 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     >
     > For subsequent runs you will be prompted to pick XCode project directory again (with the one used previously preselected). You can generate a new project or append/replace existing one. In subsequent runs, if you've set up provisioning and use Append, you can use **Build and Run** to trigger project run automatically after XCode project was generated.
 
-  1. Once the game is running on your device, you see an empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
+  1. Once the game is running on your device, you need to connect using the local IP address.<br/>Either: 
+  
+      * On your device, note the empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
+      
+      Or:
+  
+      * In your Unity Editor, navigate to: **SpatialOS** > **Launch Mobile Client** > **iOS Client** and launch mobile client with IP address field prefilled. (It's prefilled with a value from **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**.)
+  
+      > To use this workflow you need to have [ideviceinstaller](https://github.com/libimobiledevice/ideviceinstaller) and [idevicedebug](https://helpmanual.io/help/idevicedebug/) tools installed.
```

Comment:
this should be in the setup guide

Reply:
Fair enough. I'll add that to set up, but I would also prefer to keep it here - those tools aren't necessary to work, so people might skip this initially.